syncrepo.sh

```diff
@@ -219,57 +225,57 @@ syncrepo.save_config() {
 
 # Debug wrapper to trace function calls
 utils.call() {
+    [[ $# -gt 0 ]] || return 1
     local name=$1 && shift
```

Comment:
TODO: $name can just be inlined here

Reply:
This is better than the slightly less legible alternative of using "$@" directly.